dht/sharder.hh

```diff
@@ -28,28 +28,58 @@
 
 namespace dht {
 
+
+// Utilities for sharding ring partition_range:s
+
+// A ring_position range's data is divided into sub-ranges, where each sub-range's data
+// is owned by a single range. Note that multiple non-overlapping sub-ranges may map to a
```

Comment:
'owned by a single range'? Did you mean 'owned by a single shard'?

Reply:
Yes, will fix